pom.xml

```diff
@@ -236,6 +236,7 @@
         <xmltool.version>3.3</xmltool.version>
         <xml-matchers.version>0.10</xml-matchers.version>
         <zookeeper.version>3.5.5</zookeeper.version>
+        <commons-math3.version>3.6.1</commons-math3.version>
```

Comment:
I really don't want to say no, but dependencies are listed in alphabetical order...

Reply:
Placed it alphabetically in the updated patch.